templates/front/frontLayout.html.twig

```diff
@@ -0,0 +1,33 @@
+{% extends 'base.html.twig' %}
+
+{% block navbar %}
+<nav class="navbar navbar-expand navbar-light bg-light fixed-top" role="navigation">
+    <a class="navbar-brand" href="{{path('product_vehicles')}}">Mobilean</a>
+    <div class="nav-item">
+        <a class="nav-link" href="{{path('product_vehicles')}}">Nos solutions clés en main</a>
+    </div>
+    <div class="nav-item">
+        <a class="nav-link" href="{{path('product_chargingStations')}}">Nos solutions de ravitaillement</a>
+    </div>
+    <div class="nav-item">
+        <a class="nav-link" href="{{path('contact_estimate')}}">Je décris mon besoin</a>
+    </div>
+    <div class="nav-item">
+        <a class="nav-link" href="{{path('contact_partner')}}">Devenir installateur</a>
+    </div>
+    <div class="nav-item">
+        <a class="nav-link" href="{{path('information_about')}}">À propos de nous</a>
+    </div>
```

Comment:
This probably should be a list of link and not a pile of divs

Reply:
C'est une barre de navigation en Bootstrap, il demande des "div" et cela ne fonctionne pas avec des "li" malheureusement

Reply:
Nav has been created using bootstrap, hence all the divs. Using a list would need more css and would negate the use of bootstrap.